src/tla2tools.ts

```diff
@@ -101,11 +105,40 @@ export async function runSany(tlaFilePath: string): Promise<ToolProcessInfo> {
     );
 }
 
+function buildTexOptions(tlaFilePath: string, shadeComments: boolean, commentColor: number, numberLines: boolean, noPcalShade: boolean): string[] {
+    const toolArgs = ['-metadir',
+        path.dirname(tlaFilePath),
```

Comment:
`-metadir path.dirname(tlaFilePath)` seems new.  Why is it needed?

Reply:
It is used [here](https://github.com/tlaplus/tlaplus/blob/ab14a33e39c78e4c88e81b664b9a8c916b943cab/toolbox/org.lamport.tla.toolbox.tool.tla2tex/src/org/lamport/tla/toolbox/tool/tla2tex/handler/ProducePDFHandler.java#L135C1-L135C34):
```
					tla2texArgs.add("-metadir");
					tla2texArgs.add(fileToTranslate.getProject().getLocation()
							.toOSString());
```
and wanted to port it here as well.
According to [TLC](https://github.com/tlaplus/tlaplus/blob/9fc01661b60af2db3077ed5060fa514c9c9d40cf/tlatools/org.lamport.tlatools/src/tlc2/TLC.java#L1608):
```
"-metadir"
"specify the directory in which to store metadata; defaults to SPEC-directory/states if not specified"
```
I guess it's fine to omit it? I also noticed I forgot to add TLA2TeX_Output_Extension, but I'm not sure if it's really needed, what do you think?

Reply:
Sorry, I don't remember if tla2tex creates temporary files.

Reply:
Removed for now, we can add it later if needed be (doesn't seem to make any difference as far as I can tell)